CMakeLists.txt

```diff
@@ -456,6 +456,10 @@ if(FF_BUILD_MOE OR FF_BUILD_ALL_INFERENCE_EXAMPLES OR FF_BUILD_ALL_EXAMPLES)
   add_subdirectory(examples/cpp/inference/transformers)
 endif()
 
+if(FF_BUILD_ALL_EXAMPLES)
```

Comment:
since the LLAMA model is only supported in inference mode for now, we should build even if only if the `FF_BUILD_ALL_INFERENCE_EXAMPLES` flag is set to `ON`, so that we are not forced to build all the examples to run LLAMA